R/utils.R

```diff
@@ -0,0 +1,12 @@
+#' @title Send a call to function in the server and get the result
+#'
+#' @param datasources a list of \code{\link{DSConnection-class}} 
+#' objects obtained after login.
+#' @param function_call a character string that represents the call to a function with its arguments.
+#'
+#' @return \code{callAggregationMethod} returns the result from a call to a function in the server
+
+callAggregationMethod <- function(datasources, function_call) {
```

Comment:
I wonder if this function really makes you save a lot of space. If you think so, then can you integrate it as well for the other R functions ?